ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -21,5 +22,7 @@
     private CCDClaim claimData;
     private CCDCountyCourtJudgment countyCourtJudgment;
     private String countyCourtJudgmentRequestedAt;
-
+    private String defendantEmail;
+    private CCDResponse response;
+    private String respondedAt;
```

Comment:
You need to add defendantId here as well.

Reply:
no defendant id should not be here!!!!